src/User32/User32+CURSORINFOFlags.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) All contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace PInvoke
+{
+    /// <content>
+    /// Contains the <see cref="CURSORINFOFlags"/> nested type.
+    /// </content>
+    public static partial class User32
+    {
+        /// <summary>
+        /// Flags for the <see cref="CURSORINFO.flags" /> field.
+        /// </summary>
+        public enum CURSORINFOFlags
+        {
+            /// <summary>The cursor is hidden.</summary>
+            None = 0,
```

Comment:
Why not `CURSOR_HIDDEN`?

Reply:
We have a precedent here to name enum values based on the actual docs/header files, or to use `None` for 0 when the header files don't define a constant for 0. This makes it more predictable across APIs. In this case, the docs define `0` to mean hidden, but it doesn't give it a name, so we use the same name we use in all other enums for `0` where no name is defined.

Reply:
Hmmm... but I guess this isn't a `[Flags]` enum, and you're right, this really doesn't read very well as-is. I may go ahead and make your proposed change.